ocp_resources/performance_profile.py

```diff
@@ -0,0 +1,99 @@
+from ocp_resources.resource import Resource
+
+
+class PerformanceProfile(Resource):
+    """
+    PerformanceProfile object. API reference:
+    https://docs.openshift.com/container-platform/4.13/rest_api/node_apis/performanceprofile-performance-openshift-io-v2.html#performanceprofile-performance-openshift-io-v2
+
+    Args (in .spec):
+        additional_kernel_args (array of strings): Additional kernel arguments.
+        cpu (obj): Set of CPU related parameters. For example:
+            {
+                "isolated": "4-39,44-79",
+                "reserved": "0-3,40-43",
+            }
+        globally_disable_irq_load_balancing (bool): Toggles whether IRQ load balancing will be disabled for the Isolated CPU
+            set.  Defaults to "false".
+        hugepages (obj): Defines a set of huge pages related parameters. It is possible to set huge pages with multiple
+            size values at the same time.
+        machine_config_label (obj): Defines the label to add to the MachineConfigs the operator creates.
+            It has to be used in the MachineConfigSelector of the MachineConfigPool which targets this performance profile.
+            Defaults to "machineconfiguration.openshift.io/role=<same role as in NodeSelector label key>"
```

Comment:
this is done by the api, not the code here; please remove.
the docstring should include only what is handled in the code

Reply:
Done